src/scss/pages/_managers.scss

```diff
@@ -21,4 +22,19 @@
   &__table {
     width: 90%;
   }
+  // .search{
```

Comment:
If this isn't being used anymore, i would just delete it

Reply:
Good catch! Forgot to delete it when I switched to the common component~